in_toto/models/layout.py

```diff
@@ -237,6 +240,7 @@ class Step(models__common.Metablock):
   product_matchrules = attr.ib(default=attr.Factory(list))
   pubkeys = attr.ib(default=attr.Factory(list))
   expected_command = attr.ib("")
+  threshold = attr.ib(default=attr.Factory(int))
```

Comment:
I think it would make more sense to keep the default threshold value = 1.
As any step should at least be performed by one functionary.
**Already changed the default value. Tell me if you disagree.